networkx/drawing/nx_pydot.py

```diff
@@ -182,8 +168,10 @@ def to_pydot(N, strict=True):
     -----
 
     """
-    pydot = load_pydot()
-
+    try:
+        import pydotplus as pydot
+    except ImportError:
+        raise ImportError("pydotplus required for to_pydot()")
```

Comment:
Agreed... I'm for letting the standard ```ImportError``` work its magic.
Is there a reason not to import once at the top of the file? I vaguely recall there's some reason to do it inside the functions but don't recall.